src/test/resources/logback.xml

```diff
@@ -4,9 +4,9 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
         </encoder>
     </appender>
-    <logger name="cloud.quasarch" level="DEBUG" />
```

Comment:
Having this logger is handy for test debugging purposes, therefore i dont see why we should remove it

Reply:
I think it makes running tests a lot more verbose/noisy.

I can still restore it if you want to maintain this level.